var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -71,7 +71,8 @@ def install(self, spec, prefix):
             'ARCHFLAGS    = cr',
             'RANLIB       = true',
             'CC           = {0}'.format(self.spec['mpi'].mpicc),
-            'CFLAGS       = -fPIC -std=c99 -O2 %s %s %s' % (
+            'CFLAGS       = -fPIC %s -O2 %s %s %s' % (
+                '' if '%pgi' in spec else '-std=c99',
```

Comment:
Can you also replace `-fPIC` with `self.compiler.pic_flag`? Every compiler uses a different flag.